ansible/files/runtimes.json

```diff
@@ -109,6 +109,16 @@
                     "name": "action-php-v7.1"
                 }
             }
+        ],
+        "ruby": [
+            {
+                "kind": "ruby:2.5",
+                "default": true,
+                "deprecated": false,
+                "image": {
```

Comment:
with #3680, the image prefix and tag should be specified explicitly (to `openwhisk` and `latest` respectively).

Reply:
made a change as a result of #3680

Reply:
changed `prefix` and `name` temporarily to workaround the build failure. (thank you @rabbah for the quick advice to fix this)